src/nxt_main_process.c

```diff
@@ -190,18 +190,6 @@ static nxt_conf_map_t  nxt_python_app_conf[] = {
         offsetof(nxt_common_app_conf_t, u.python.path),
     },
 
-    {
-        nxt_string("module"),
-        NXT_CONF_MAP_STR,
-        offsetof(nxt_common_app_conf_t, u.python.module),
-    },
-
-    {
-        nxt_string("callable"),
-        NXT_CONF_MAP_CSTRZ,
-        offsetof(nxt_common_app_conf_t, u.python.callable),
-    },
-
```

Comment:
I didn't realize:  shouldn't this be removed in the first commit?  The one that removes unused code.